README.md

```diff
@@ -1,2 +1,34 @@
 # positioning-floating-elements
 This assignment consists of making a clone of a New York Times article page.
```

Comment:
- You do not need to say that this was an assignment from a school if you do not want to. I believe it is better to say what you showcased in this project, like: 'We cloned the New York Times Pages using grids, flexbox and ...'